modules/swagger-codegen/src/main/resources/JavaPlayFramework/formParams.mustache

```diff
@@ -0,0 +1 @@
+{{#isFormParam}}{{#notFile}}@ApiParam(value = "{{{description}}}"{{#required}}, required=true{{/required}} {{#allowableValues}}, allowableValues="{{#enumVars}}{{{name}}}{{^-last}}, {{/-last}}{{#-last}}{{/-last}}{{/enumVars}}"{{/allowableValues}}{{#defaultValue}}, defaultValue="{{{defaultValue}}}"{{/defaultValue}}) @RequestPart(value="{{paramName}}"{{#required}}, required=true{{/required}}{{^required}}, required=false{{/required}})  {{{dataType}}} {{paramName}}{{/notFile}}{{#isFile}}@ApiParam(value = "file detail") @RequestPart("file") MultipartFile {{baseName}}{{/isFile}}{{/isFormParam}}
```

Comment:
Please use `{{baseName}}` instead of `{{paramName}}` in annotations which refer to the on-the-wire parameter name.
`{{paramName}}` is the sanitized version of the parameter name, `{{baseName}}` how it was mentioned in the API definition. (See #4898 for a lot of cleanup regarding this.)